src/components/App.js

```diff
@@ -37,7 +38,7 @@ class App extends Component {
               exact path={routes.SESSION}
               component={() => <SessionPage />}
             />
-
+            <LogOutFAB />
```

Comment:
This should probably only be visible if the user is logged in.

Reply:
I had a few 🍻 when I was working on this 😬 This is supposed to only be hidden. I think that I need to stop using the `hidden` attribute and use conditional rendering with a ternary return. I'll switch that out soon. #36